### ISSUE TITLE
Deploying new canisters is rejected 

### ISSUE THREAD

**Issue by mikhail-turilin-dfinity** [dfinity/sdk#2123]
**TLDR:**
Newly created canisters generate `Error: The replica returned an HTTP Error: Http Error: status 403 Forbidden, content type "application/cbor", content: Requested canister rejected the message` at deployment.

**Prerequisites:**
- Install dfx and Node 16.x (not 17.x) as described [here](https://smartcontracts.org/docs/quickstart/1-quickstart.html)
- Acquire IC cycles as described in [this guide](https://smartcontracts.org/docs/quickstart/4-quickstart.html)

**Scenario:**
1. Create a new directory and add files (html, css, js, dfx.json) as described in [Hosting a Static Website on the Internet Computer](https://smartcontracts.org/docs/samples/host-a-website.html)
2. Run a command `dfx deploy --network ic`

**Expected result:**
The canister is deployed with a message:
```
Deploying all canisters.
All canisters have already been created.
Building canisters...
Installing canisters...
Installing code for canister website, with canister_id osjqh-4aaaa-aaaai-acgra-cai
Uploading assets to asset canister...
Starting batch.
Staging contents of new and changed assets:
  /index.html 1/1 (490 bytes)
  /page-2.html 1/1 (356 bytes)
  /script.js 1/1 (114 bytes)
  /style.css 1/1 (105 bytes)
  /index.html (gzip) 1/1 (316 bytes)
  /page-2.html (gzip) 1/1 (254 bytes)
Committing batch.
Deployed canisters.
```
**Observed result:**
```
Error: The replica returned an HTTP Error: Http Error: status 403 Forbidden, content type "application/cbor", content: Requested canister rejected the message
```

**Temporary fix:**
Run the command at the root of the project:
`% dfx canister --wallet "$(dfx identity --network ic get-wallet)" --network ic update-settings --all --add-controller "$(dfx identity get-principal)"`

Then you can attempt to deploy again with `dfx deploy --network ic`
